CorsixTH/Lua/config_finder.lua

```diff
@@ -105,8 +105,8 @@ local config_defaults = {
   announcement_volume = 0.5,
   play_music = true,
   music_volume = 0.5,
-  prevent_edge_scrolling = false,
-  capture_mouse = true,
+  prevent_edge_scrolling = true,
+  capture_mouse = false,
```

Comment:
Changing these settings no longer matches with the default config file for the windows installer.

Reply:
Where is the default config file for the Windows installer? I couldn't find any settings related to mouse scrolling or capturing.

Reply:
https://github.com/CorsixTH/CorsixTH/blob/master/WindowsInstaller/config_template.txt